gapis/resolve/framebuffer_attachment.go

```diff
@@ -31,6 +31,15 @@ import (
 )
 
 func FramebufferAttachments(ctx context.Context, p *path.FramebufferAttachments, r *path.ResolveConfig) (interface{}, error) {
+	cmd, err := Cmd(ctx, p.After, r)
+	if err != nil {
+		return nil, err
+	}
+
+	if !cmd.CmdFlags().IsExecutedDraw() && !cmd.CmdFlags().IsEndOfFrame() {
+		return nil, &service.ErrDataUnavailable{Reason: messages.ErrNotADrawCall()}
+	}
+
```

Comment:
This should not be here. The resolvable should return this error instead. I know we used to have this functionality, at least in GLES, where it would give a good error message (such as "there's no framebufer, something, something").

It should be the resolvable that determines when an FB is available.

Reply:
Non-draw/non-present calls technically do have resolvable framebuffer attachments. Unless you mean I physically copy-and-paste those line to the resolve functon.

Reply:
No, my point is exactly that we should still show them in those cases. I am pointing out that it should be the act of resolving that decides whether a FB is available and applicable at a specific command. It should not be the API-agnostic part of the code that decides.

The issue is when you select a command where there isn't an FB, the error is not nice and we should hide the thumbnails.

Reply:
I see. Well, that's easy to fix: I just get rid of these lines. The UI code responds the same way regardless of error.